### DISCUSSION TITLE
Could not find a declaration file for module 'prettier'.

### DISCUSSION THREAD

**Discussion by patrickpannn** [renovatebot/renovate#24308] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Cloud (bitbucket.org)

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

When I run npm run build and it throws these errors. I have already installed @types/prettier but still not working. "Could not find a declaration file for module 'prettier'." Any help?

### Relevant debug logs

<details><summary>Logs</summary>

```
"Could not find a declaration file for module 'prettier'."```

</details>


### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
Why are you running "npm run build"?

**Reply by patrickpannn** [reply]
Wanted to transpile to js to run faster

**Reply by rarkins** [reply]
No need, install the npm package

**Reply by patrickpannn** [reply]
You mean just run npm Install and then execute the typescript files?

**Reply by rarkins** [reply, marked as answer]
Please see https://docs.renovatebot.com/#ways-to-run-renovate

There is no need to run from source

**Comment by rarkins** [comment]
Also this project uses pnpm, not npm
